gaia-button.js

```diff
@@ -24,6 +24,24 @@ module.exports = component.register('gaia-button', {
     this.setupShadowRoot();
     this.setAttribute('role', 'button');
     this.tabIndex = 0;
+    if (this.keyboardEnabled) {
```

Comment:
Do we want this feature by default? Is there a significant overhead to to binding to these events?